src/modules/tools/endstops/Endstops.cpp

```diff
@@ -1085,6 +1098,35 @@ void Endstops::on_gcode_received(void *argument)
                 gcode->stream->printf(" will take effect next home\n");
                 break;
 
+            case 211: { //Enable or disable endstop(s) hard limit
+                if(gcode->has_letter('S')) { //Alter endstop behavior    
+                        for(auto& e : endstops) {
+                            if(gcode->has_letter(e->axis)) {
+                                if(e->is_max_stop == gcode->get_value(e->axis)) {  //Are we configuring a min or max endstop?
```

Comment:
You can continue to do this in your fork if you wish. However it will not be accepted as a PR if it is continued to be done this way.. Sorry. There should be no link to if it is min or max, this needs to be explicitly set in the M211 syntax by one of the methods I have already suggested